src/pages/en/guides/imports.md

```diff
@@ -210,3 +210,44 @@ const data = JSON.parse(json);
 
 <span>Version: {data.version}</span>
 ```
+
+## Extending file type support
+
+With **Vite** and compatible **Rollup** plugins, you can import file types which aren't natively supported by Astro. Learn where to find the plugins you need in the [Finding Plugins](https://vitejs.dev/guide/using-plugins.html#finding-plugins) section of the Vite Documentation.
+
+
+ :::note[plugin configuration]
+Refer to your plugin's documentation for configuration options, and how to correctly install it.
+:::
+
+### Example: YAML support
+
+The YAML (`.yml`) data format isn't natively supported by Astro, but you can add support for it by using a compatible Rollup plugin like [`@rollup/plugin-yaml`](https://www.npmjs.com/package/@rollup/plugin-yaml):
+
+1. Install `@rollup/plugin-yaml`:
+
+    ```bash
+    npm install @rollup/plugin-yaml --save-dev
+    ```
+
+2. Import the plugin in your `astro.config.mjs` and add it to the Vite plugins array:
+
+    ```js title="astro.config.mjs" ins={2,5-7}
+    import { defineConfig } from 'astro/config';
+    import yaml from '@rollup/plugin-yaml';
+
+    export default defineConfig({
+      vite: {
+        plugins: [yaml()]
+      }
+    });
+    ```
+
+
+3. Finally, you can import YAML data into an Astro file using an `import` statement:
+
+    ```astro title="src/components/Component.astro"
+    ---
+    import yml from './data.yml';
+    ---
+    ```
```

Comment:
In which case, I’d probably simplify down to just the import statement, no filename and code fences:

````suggestion
    ```js
    import yml from './data.yml';
    ```
````